plaid/payment.go

```diff
@@ -159,25 +167,41 @@ func (c *Client) CreatePayment(
 	amount PaymentAmount,
 	schedule *PaymentSchedule,
 ) (resp CreatePaymentResponse, err error) {
+	return c.CreatePaymentWithOptions(recipientID, reference, amount, schedule, nil)
+}
+
+func (c *Client) CreatePaymentWithOptions(
+	recipientID string,
+	reference string,
+	amount PaymentAmount,
+	schedule *PaymentSchedule,
+	options *PaymentOptions,
```

Comment:
It seems a little odd to have options be optional (pointer) if the function is `CreatePaymentWithOptions`?

I think it may make sense to move the shared logic into a helper function instead, or send an empty options {} to the server? This would make the interface more consistent with transactions at least:
https://github.com/plaid/plaid-go/blob/master/plaid/transactions.go#L99